### PR TITLE
Update Settings docs for 2.6.0 RC1

### PR DESCRIPTION
**Pull request by nmaggioni**

### CODE REVIEW

File: docs/Settings.md [hunk 1]
```diff
@@ -125,6 +128,7 @@
 | fw_p_yaw | 6 | Fixed-wing rate stabilisation P-gain for YAW |
 | fw_reference_airspeed | 1000 | Reference airspeed. Set this to airspeed at which PIDs were tuned. Usually should be set to cruise airspeed. Also used for coordinated turn calculation if airspeed sensor is not present. |
 | fw_tpa_time_constant | 0 | TPA smoothing and delay time constant to reflect non-instant speed/throttle response of the plane. Planes with low thrust/weight ratio generally need higher time constant. Default is zero for compatibility with old setups |
+| fw_turn_assist_pitch_gain | 1 | Gain required to keep constant pitch angle during coordinated turns (in TURN_ASSIST mode). Value significantly different from 1.0 indicates a problem with the airspeed calibration (if present) or value of `fw_reference_airspeed` parameter |
```
Comment (avsaase) [inline docs/Settings.md:131]: This comes from #6101, but in the discussion there it became clear that flying wings most likely all need a value lower than 1.  Can this text still be changed, or do I need to open another PR for that?

Comment (nmaggioni) [reply docs/Settings.md:131]: These changes are autogenerated from [settings.yaml][1], you'd need to change the text there first.

[1]: https://github.com/iNavFlight/inav/blob/master/src/main/fc/settings.yaml
